problems.txt

```diff
@@ -0,0 +1,96 @@
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.177] [INFO] knj_restclient_log - No any service is bound for this application.
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.179] [ERROR] knj_restclient_log - No ingress URL is set, please set "ingressURL" in your configuration json file
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - Failed to get /home/sam/w/sn/strong-supervisor/bin//package.json
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - NPM Packages discovered:
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->appmetrics: ^3.x
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->appmetrics-dash: ^3.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->async: ^2.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->debug: ^2.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->dotenv: ^1.0.0
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->ibmapm: ^1.0.8
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->lodash: ^4.14.1
+2017-07-06T17:31:59.184Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - -->posix-getopt: ^1.1.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-cluster-control: ^2.0.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-control-channel: ^2.0.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-log-transformer: ^1.0.0
+2017-07-06T17:31:59.185Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-npm-ls: ^1.0.0
+2017-07-06T17:31:59.186Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-statsd: ^2.0.0
+2017-07-06T17:31:59.186Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-trace: ^1.2.0
+2017-07-06T17:31:59.187Z pid:20915 worker:1 [2017-07-06 10:31:59.181] [INFO] knj_log - -->strong-url-defaults: ^1.2.0
+2017-07-06T17:31:59.187Z pid:20915 worker:1 [2017-07-06 10:31:59.182] [ERROR] knj_restclient_log - No ingress URL is set, please set "ingressURL" in your configuration json file
+2017-07-06T17:31:59.187Z pid:20915 worker:1 [2017-07-06 10:31:59.182] [INFO] knj_log - JsonSender: VCAP_APPLICATION env variable not found, it is not set in non-bluemix env
+2017-07-06T17:31:59.188Z pid:20915 worker:1 [2017-07-06 10:31:59.186] [ERROR] knj_restclient_log - No ingress URL is set, please set "ingressURL" in your configuration json file
+2017-07-06T17:31:59.188Z pid:20915 worker:1 [2017-07-06 10:31:59.186] [WARN] knj_restclient_log - sendMetrics payload is not complete, must have: resourceID, dimensions and metrics
```

Comment:
Is this an internal bug? It doesn't look like something a user can do anything about.